lib/ex_unit/lib/ex_unit/on_exit_handler.ex

```diff
@@ -54,6 +54,20 @@ defmodule ExUnit.OnExitHandler do
         {runner_pid, runner_monitor, state || error}
       {:DOWN, ^runner_monitor, :process, ^runner_pid, error} ->
         {nil, nil, state || {{:EXIT, runner_pid}, error, []}}
+    after
+      timeout ->
+        stacktrace =
+          try do
+            Process.info(runner_pid, :current_stacktrace)
+          catch
+            _, _ -> []
```

Comment:
Oh, indeed. That's what you get when you blindly copy code from [another place](https://github.com/elixir-lang/elixir/blob/85d3d8171482ec041f840c25f16f64e85acde5ff/lib/ex_unit/lib/ex_unit/runner.ex#L272-L281). 😄
Will fix that in there as well.